irnerio-mosaico/ontology/mmdio/.htaccess

```diff
@@ -0,0 +1 @@
+RewriteRule ^$ https://github.com/irnerio-mosaico-opendata/mmdio/raw/main/docs/current/mmdio.html [R=302,L]
```

Comment:
Looks like you do have pages enabled.  As @dgarijo mentioned, this link is to the raw html.  You probably want this:
```suggestion
RewriteRule ^$ https://irnerio-mosaico-opendata.github.io/mmdio/docs/current/mmdio.html [R=302,L]
```

Reply:
Thank you so much!! It's working correctly